src/lib/optimizer/strategy/disjunction_to_union_rule.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "abstract_rule.hpp"
+
+namespace opossum {
+
+/**
+ * This rule turns PredicateNodes with disjunctive chains (e.g., `PredicateNode(a OR b OR c)`) as their scan expression
+ *    into n-1 consecutive UnionNodes that combine the results of n PredicateNodes with simple expressions.
```

Comment:
What is a "simple expression"? Those could be arbitrarily complex, too. Leave out the "simple"?